pmix/borrow.py

```diff
@@ -106,7 +105,9 @@ def merge_translation_file(merge: List[str], translation_dict: TranslationDict,
         pathlib.Path(outpath).mkdir(parents=True, exist_ok=True)
     for path in merge:
         xlsform = Xlsform(path)
-        xlsform.add_language(add)
+        add_list = [add] if isinstance(add, str) else add
+        for language in add_list:
+            xlsform.add_language(language)
```

Comment:
This code is not needed. The `add_language` routine can handle a string or a sequence.

Reply:
@jkpr My IDE was pointing out that `List[str]` was not the same thing as str. And it pointed out that `add_language()` accepts a string. It is probably picking up `add_language()`'s docstring annotation. We should change that to `List[str]` as well probably then.

Reply:
i. **Warning**
Not getting a warning in Pycharm now. Either this is due to me updating it, or due to the fact (more likely) that I started off using an old branch and rebase/refactored later.

ii. **add_language()**
Actually it does not support a list: https://github.com/PMA-2020/pmix/blob/develop/pmix/xlsform.py#L87
```
    def add_language(self, language):
        """Add appropriate language columns to an Xlsform.

        Args:
            language (str): The language to add to all relevant sheets.
        """
        for sheet in self:
            sheet.add_language(language)
```